src/main/java/seedu/address/model/powercard/Question.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.powercard;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Powercard's Question in the deck.
+ */
+public class Question {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Questions should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Sorry but I am not really familiar with regular expressions. What does this REGEX do? I dont think this regex is the same regex in the Address class, but the javadoc comment for both regex are the same